flytekit/tools/repo.py

```diff
@@ -91,6 +91,10 @@ def package(
 
         # If Fast serialization is enabled, then an archive is also created and packaged
         if fast:
+            # If output exists and is a path within source, delete it so as to not re-bundle it again.
+            if os.path.abspath(output).startswith(os.path.abspath(source)) and os.path.exists(output):
```

Comment:
does this ever get triggered?

Reply:
If you do `pyflyte --pkgs core package --image core:v1 --fast --force` again and again, then yes

Reply:
This is the original issue of `.tar.gz` doubling in size every time it is run

Reply:
I am keeping this and resolving the comment